USAGE.md

```diff
@@ -0,0 +1,104 @@
+# Amazon Alexa Smart Home Skill for openHAB 2
+
+Amazon certified openHAB2 as a [smarthome skill for Alexa](https://www.amazon.com/openHAB-Foundation/dp/B01MTY7Z5L).
+This page describes how to use the skill certified by Amazon.
+This skill allows you to connect your openHAB setup through the [myopenHAB.org](http://myopenHAB.org) cloud service to Amazon Alexa.
+(See [README.md](https://github.com/openhab/openhab-alexa/blob/master/README.md) for other setup options and development information.)
+
+With this skill you can control items that respond to on/off and numeric commands.
+The Skill as well has limited support for thermostats.
+The skill is supported for English (U.S.), English (U.K.), and German languages. 
+
+## General Configuration Instructions
+
+### Requirements
+
+* [openHAB Cloud Connector](http://docs.openhab.org/addons/io/openhabcloud/readme.html) configured using myopenHAB.org
+* Amazon account
+* Amazon Echo, Amazon Echo Dot or compatible Alexa speaker or solution (e.g. [Reverb](https://reverb.ai/))
+
+### 1. Item Configuration
+
+You have to tag your items to make them useable by Alexa.
+See [Tagging](http://docs.openhab.org/configuration/items.html#tagging) for more details.
```

Comment:
And I would remove this or change it to
```See [Tagging](http://docs.openhab.org/configuration/items.html#tagging) for more information on how tags are used in openHAB.```

Reply:
WIll remove it. Tagging right now hassn't much informations.